app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -1,29 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="http://schemas.android.com/tools">
 
-    <string name="default_location_label">São Paulo</string>
```

Comment:
Is this a reversion? The default location for Portuguese locales is changed from São Paulo to "New York" (generic default).